### DISCUSSION TITLE
[求助]Minecraft联机遇到问题

### DISCUSSION THREAD

**Discussion by Eeeeepiphany** [share121/inter-knot#4728] [委托]
版本1.16.5forge，我正版登录，对方离线登录，带模组，采用樱花穿透并关闭正版验证，对方无法进入游戏但可以根据我的模组包开单人游戏且双方单人游戏所有模组均生效，图为模组包内容
![Screenshot_2024_0910_104517](https://github.com/user-attachments/assets/c088a413-fbf4-4997-98f1-36fa2ea6d52d)

**Comment by Akanyi** [comment]
可以尝试利用在本地创建服务器的方式联机，使用樱花穿透穿透服务器的端口，然后两个人同时加入服务器，你可以直接用本地加入，也可以用樱花穿透加入，如果不行请检查你的网络，有很大概率是网络问题。
